shared/src/packages/orders/libs/types/order-entity.type.ts

```diff
@@ -0,0 +1,17 @@
+import { type OrderStatus } from './order-status.type.js';
+
+type OrderEntity = {
+  id: number;
+  price: number;
+  scheduledTime: string;
+  startPoint: string;
+  endPoint: string;
+  status: OrderStatus;
+  userId: number;
```

Comment:
it's nullable, because we'll allow to make orders without registration

Reply:
got it, fixed